lib/takeaway.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'menu'
+require_relative 'bill_calculator'
+require_relative 'text'
+class Takeaway
+
+  attr_reader :order, :cost, :get_bill
+
+  def initialize
+    @menu = Menu.new
+    @order = []
+    @get_bill = BillCalculator.new
+  end
+
+  def display_menu
+    @menu.show_menu
+  end
+
+  def select_dish(dish)
+    fail "Selection unavailable" if @menu.on_menu?(dish) == false
+
```

Comment:
I think you could refactor this to use an 'unless' instead, which would increase readability and not require the '== false' statement.